src/test/ccl/ut_cclut_mock_table.inc

```diff
@@ -1208,13 +1207,15 @@ subroutine test_cclutAddMockData_too_many_columns(null)
     call cclutDefineMockTable("sample_table", "sample_table_id|sample_table_text|sample_table_date", "f8|vc|dq8")
     call cclutCreateMockTable("sample_table")
 
+    call cclutAssertVcOperator(CURREF, "cclutAssertVcOperator check", "this is a test string", "regexplike", "test")
```

Comment:
Does this serve a purpose for the mock_table tests?  It feels like this should belong in the assertion unit tests.

Reply:
lingering debug code. removed.